lib/ui/painting/image_encoding.cc

```diff
@@ -65,15 +65,24 @@ void InvokeDataCallback(std::unique_ptr<DartPersistentValue> callback,
 
 sk_sp<SkImage> ConvertToRasterUsingResourceContext(
     sk_sp<SkImage> image,
-    GrDirectContext* resource_context) {
+    fml::WeakPtr<GrDirectContext> resource_context,
+    const std::shared_ptr<const fml::SyncSwitch>& gpu_disable_sync_switch) {
```

Comment:
s/gpu_disable_sync_switch/is_gpu_disabled_sync_switch

That way the "SetIfTrue" makes sense.

Reply:
Done